src/js/otus/queries.tsx

```diff
@@ -28,6 +30,8 @@ export const OTUQueryKeys = {
     infiniteList: (filters: Array<string | number | boolean>) => ["OTU", "list", "infinite", ...filters] as const,
     details: () => ["OTU", "details"] as const,
     detail: (id: string) => ["OTU", "detail", id] as const,
+    histories: () => ["OTU", "details", "history"] as const,
+    history: (id: string) => ["OTU", "detail", "history", id] as const,
```

Comment:
Hmm I think there is bug here: 
```suggestion
    details: () => ["OTU", "detail"] as const,
    detail: (id: string) => ["OTU", "detail", id] as const,
    histories: () => ["OTU", "detail", "history"] as const,
    history: (id: string) => ["OTU", "detail", "history", id] as const,
```
I think the pluralized keys would break the query partial query matching during invalidation